expyre/cli/cli.py

```diff
@@ -31,8 +32,9 @@ def _get_jobs(**kwargs):
 @click.option("--name", "-n", help="comma separated list of regexps for entire job name")
 @click.option("--status", "-s", help="comma separated list of status values to include")
 @click.option("--system", "-S", help="comma separated list of regexps for entire system name")
+@click.option("--long_output", "-l", is_flag=True, help="long format output")
```

Comment:
Could we consistently use dashes (`long-output`)? That's the default in workflow, so will be nicer from user perspective.

Reply:
Yes, sorry, I default to underscores, but no one else does.